src/components/TrackGameOne.js

```diff
@@ -0,0 +1,38 @@
+import Input from './Input.js';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+TrackGameOne.propTypes = {
+  onHandleChange: PropTypes.func,
+  gameData: PropTypes.string
+};
+
+export default function TrackGameOne({ onHandleChange, gameData }) {
+  return (
+    <>
```

Comment:
Fragments are not necessary here since you have the `GameNameContainer` component as a parent.